### ISSUE TITLE
Lint the outer pipe as well in a doubly-nested one-call pipe

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#2488]
```r
expect_lint("x %>% inner_join(y %>% filter(is_treatment))", lint_msg, linter)
```

This should return two lints:

```r
x %>% inner_join(.) # doesn't lint now
y %>% filter(is_treatment) # lints as expected
```

**Comment by AshesITR** [comment]
What linter are you referring to?

**Comment by MichaelChirico** [comment]
Edited `linter` -> `one_call_pipe_linter()` to be specific (I think "one-call pipe" in the issue title is enough there).
